pkg/security/secl/model/bpf_maps_generator/bpf_maps_generator.go

```diff
@@ -108,3 +115,14 @@ func main() {
 		panic(err)
 	}
 }
+
+func checkDuplicated(entries []mapEntry) {
+	trimmedNames := make(map[string]bool, len(entries))
+
+	for _, entry := range entries {
+		if trimmedNames[entry.TrimmedName] {
+			fmt.Printf("CWS warning: the trimmed map name `%s`(`%s`) is not unique\n", entry.TrimmedName, entry.Name)
```

Comment:
Should we return an error instead ? Otherwise, we may simply miss the warning

Reply:
I'm not 100% sure what exactly we should no if there is a collision for now, I plan to fix the collision in another PR and turn this into an error there

Reply:
Ok